### ISSUE TITLE
evaluation code

### ISSUE THREAD

**Issue by kargarisaac** [traveller59/second.pytorch#59]
Hi,
I tested the pre-trained model on first 200 sample of kitti dataset but the results are not good:
Car AP@0.70, 0.70, 0.70:
bbox AP:0.01, 0.03, 0.04
bev  AP:0.02, 0.01, 0.01
3d   AP:0.01, 0.01, 0.01
aos  AP:0.01, 0.02, 0.02
Car AP@0.70, 0.50, 0.50:
bbox AP:0.01, 0.03, 0.04
bev  AP:0.02, 0.01, 0.01
3d   AP:0.01, 0.01, 0.01
aos  AP:0.01, 0.02, 0.02

Car coco AP@0.50:0.05:0.95:
bbox AP:0.13, 0.19, 0.22
bev  AP:0.01, 0.01, 0.01
3d   AP:0.01, 0.00, 0.00
aos  AP:0.07, 0.10, 0.11

I saw the get_mAP_v2 code. There is 11 in denominator. What is it? Is there anything that I need to change?

def get_mAP_v2(prec):
    sums = 0
    for i in range(0, prec.shape[-1], 4):
        sums = sums + prec[..., i]
    return sums / 11 * 100

**Comment by traveller59** [comment]
are you using ubuntu 18.04? There is a problem of SparseConvNet in 18.04 with gcc 6+ in my environment. you may need to use gcc-5.

**Comment by kargarisaac** [comment]
@traveller59 
No, I use Ubuntu 16.04. My code runs without any error. Is it possible to run it without error and the results be awful? 
- Is there any file or parameter I need to change if I want to train the model on my own data set?
- The memory usage is very high. Is the inference time you reported in the paper for one single network (cars for example) or both networks for cars and people?
- How can I use SECOND (small)? Is there any pre-trained model for that too?

Thank you

**Comment by traveller59** [comment]
I haven't test model in subset of val set but I think your result is wrong, consider using the visualization tool in this project to check result. Maybe your sparseconvnet isn't be correctly compiled.
1. you need to prepare kitti format dataset. I am considering to reduce dataset format requirement.
2. that is car inference speed. people is slower (when using 0.1, 0.1, 0.3 voxel size) or bad performance (0.2, 0.2, 0.3 voxel size)
3. the car_tiny is small pretrained model.

**Comment by Petros626** [comment]
@kargarisaac Hey could you help me in general with my KITTI evaluation? I'm trying to bring my model predictions in the right format to evaluate, but my values are also very low.
